.github/workflows/sdk-cli-perf-monitor-test.yml

```diff
@@ -0,0 +1,80 @@
+# execute tests in src/promptflow/tests/sdk_cli_azure_test, src/promptflow/tests/sdk_cli_test with mark perf_monitor_test
+# when pull_request triggers, use live mode
+
+name: sdk-cli-perf-monitor-test
+
+
+on:
+  pull_request:
+    branches:
+      - main
+
+  schedule:
+    - cron: "0 */6 * * *"  # Run every 6 hours
+
+  workflow_dispatch:
+
+
+env:
+  packageSetupType: promptflow_with_extra
+  testWorkingDirectory: ${{ github.workspace }}/src/promptflow
+  PYTHONPATH: ${{ github.workspace }}/src/promptflow
+  IS_IN_CI_PIPELINE: "true"
+
+
+jobs:
+  sdk_cli_perf_monitor_test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+
+      - name: Display and Set Environment Variables
+        run: |
+          export pyVersion="3.9";
+          env | sort >> $GITHUB_OUTPUT
+        id: display_env
+        shell: bash -el {0}
+
+      - name: Python Setup - ${{ matrix.os }} - Python Version ${{ steps.display_env.outputs.pyVersion }}
+        uses: "./.github/actions/step_create_python_environment"
+        with:
+          pythonVersion: ${{ steps.display_env.outputs.pyVersion }}
+
+      - name: Build wheel
+        uses: "./.github/actions/step_sdk_setup"
+        with:
+          setupType: ${{ env.packageSetupType }}
+          scriptPath: ${{ env.testWorkingDirectory }}
+
+      - name: Get number of CPU cores
+        uses: SimenB/github-actions-cpu-cores@v1
+        id: cpu-cores
+
+      - name: Azure login
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: generate live test resources (non pull_request workflow)
+        uses: "./.github/actions/step_generate_configs"
+        with:
+          targetFolder: ${{ env.testWorkingDirectory }}
+
+      - name: Run Test
+        shell: pwsh
+        working-directory: ${{ env.testWorkingDirectory }}
+        run: |
+          gci env:* | sort-object name
+          python "../../scripts/building/run_coverage_tests.py" `
+            -p promptflow `
+            -t ${{ github.workspace }}/src/promptflow/tests/sdk_cli_azure_test ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test `
+            -l eastus `
+            -m "perf_monitor_test" `
+            -n ${{ steps.cpu-cores.outputs.count }} `
+            --coverage-config ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test/.coveragerc
```

Comment:
remove from -n

Reply:
Is this to filter out directories that do not need to be searched or directories that need to be searched when searching for tests?